src/main/java/com/exadel/discountwebapp/exception/exception/client/IncorrectFilterInputException.java

```diff
@@ -0,0 +1,14 @@
+package com.exadel.discountwebapp.exception.exception.client;
+
+import com.exadel.discountwebapp.exception.exception.base.ClientBaseException;
+import lombok.Getter;
+
+@Getter
+public class IncorrectFilterInputException extends ClientBaseException {
+
+    private static final String MESSAGE_PATTERN = "Could not find in class %s field %s";
+
+    public IncorrectFilterInputException(String className, String fieldName, Object value) {
```

Comment:
you can use Class name instead of string className for more flexibility for the future (in case entity will be renamed)

Class clazz

and then use clazz.getSimpleName

Reply:
done